<launch>

    <!-- LIO-SAM -->
    <param name="lidar_topic" type="string" value="/lio_sam/mapping/cloud_registered_local" />
    <param name="odom_topic" type="string" value="/lio_sam/mapping/odometry" />
    <param name="odom_frame" type="string" value="odom" />
    <param name="lidar_frame" type="string" value="base_link" />

    <!-- BotanicGarden数据集用Bow3d回环检测，应该测线束设置成32，若为16会出现较多误检测 -->
    <!-- <param name="nScans" type="int" value="16" /> -->
    <param name="nScans" type="int" value="32" />
    <!-- <param name="nScans" type="int" value="64" /> -->

    <!-- <param name="sensor_type" type="string" value="Ouster" /> -->

    <!-- nodes -->
    <node pkg="lidar_slam_loop_test" type="bow3d_online" name="bow3d_online" output="screen" />

    <!-- visulaization -->
    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz"
            args="-d $(find lidar_slam_loop_test)/rviz_cfg/loop_test.rviz" />
    </group>

</launch>